src/AutoRest.CSharp/Common/AutoRest/Plugins/Configuration.cs

```diff
@@ -132,8 +145,52 @@ public enum UnreferencedTypesHandlingOption
             _intrinsicTypesToTreatEmptyStringAsNull.UnionWith(additionalIntrinsicTypesToTreatEmptyStringAsNull);
         }
 
+        private static string? DownloadLatestContract(string projectFolder)
+        {
+            if (AzureArm || Generation1ConvenienceClient)
+            {
+                return null;
+            }
+
+            // projectFolder = "D:\\GIT\\azure-sdk-for-net\\sdk\\analysisservices\\Azure.ResourceManager.Analysis\\src";
+            int sdkFolderIndex = projectFolder.LastIndexOf("sdk", StringComparison.InvariantCultureIgnoreCase);
```

Comment:
```suggestion
            int sdkFolderIndex = projectFolder.LastIndexOf("\\sdk\\", StringComparison.InvariantCultureIgnoreCase);
```

Reply:
Better to add `\` before and after to avoid searching out the work include `sdk` in it.